internal/boxcli/services.go

```diff
@@ -232,7 +244,5 @@ func startProcessManager(
 		cmd.Context(),
 		servicesFlags.runInCurrentShell,
 		args,
-		flags.background,
-		flags.processComposeFile,
```

Comment:
`flags.processComposeFile` can be removed safely (since it is passed as a devbox option), but I think we want to keep background here.